gradle.properties

```diff
@@ -2,6 +2,6 @@ kotlin.code.style=official
 android.useAndroidX=true
 org.gradle.jvmargs=-Xmx4g
 
-LIBRARY_VERSION=0.2.4
+LIBRARY_VERSION=0.1.0
```

Comment:
Should we remove the `0.2.4` artifacts from the repo?

Reply:
yes, removed